source/development/misra.rst

```diff
@@ -279,31 +279,38 @@ following the format:
 
     ---
 
-    # The tag should always be the same name of the file, start with MDR followed # by the record
-    ID. tag: MDR1
+    # The tag should always be the same name of the file, start with MDR followed
+    # by the record ID.
+    tag: MDR1
 
-    # Optionally, list a deviation permits used as a base for the deviation. permits:
+    # Optionally, list a deviation permits used as a base for the deviation.
+    permits:
       - "MDP1"
       - "MPD2"
 
-    # List the guidelines that are being violated. guidelines:
+    # List the guidelines that are being violated.
+    guidelines:
       - "R2.5"
       - "D4.4"
 
-    # Summarize the violation and its context. Optional if a permit is selected. use_case: >
+    # Summarize the violation and its context. Optional if a permit is selected.
+    use_case: >
         This describes the records use case.
 
-    # List one or more of the allowed justification items. Optional if a permit is # selected.
+    # List one or more of the allowed justification items. Optional if a permit is selected.
     reasons:
       - Code quality (usability).
       - ....
 
-    # Detail the use cases and reasons listed above. description: >
+    # Detail the use cases and reasons listed above.
+    description: >
         Provide a detailed description of the record.
 
-    # Assess how the risks described in the guideline's rationale affect this violation # and
-    describe how they are managed or mitigated in this violation. In case a # permit is selected,
-    detail point by point how the deviation fulfils the # permit's requirements. risk: >
+    # Assess how the risks described in the guideline's rationale affect this violation
+    # and describe how they are managed or mitigated in this violation. In case a
+    # permit is selected, detail point by point how the deviation fulfils the
+    # permit's requirements.
```

Comment:
```suggestion
    # Assess how the risks described in the guideline's rationale affect this violation and
    # describe how they are managed or mitigated in this violation. In case a permit is selected,
    # detail point by point how the deviation fulfills the permit's requirements.
```